account_consolidation/models/account_move.py

```diff
@@ -14,13 +14,19 @@ class AccountMove(models.Model):
         string='Consolidated from Company',
         readonly=True
     )
+    # Technical field to mark consolidation entries that must be reverted on
+    # the next run (previously defined in account_reversal)
+    to_be_reversed = fields.Boolean(
```

Comment:
There's a UX issue with the existing field in Odoo. Consolidation entries appear like that:
![image](https://user-images.githubusercontent.com/778828/58185529-927a7580-7cb3-11e9-8704-c6fd479daa92.png)

It's a bit confusing that odoo's checkbox appears False while this field is True but not displayed.

Suggestion:
Add the `to_be_reversed` field in the view (below reverse automatically), and add the following slots here:
```
string='To be reversed on next consolidation'
help='Entries generated by the consolidation wizard are automatically reversed on the next run of the consolidation wizard'
```

And make sure the field is only displayed to `group_consolidation_manager`

Reply:
I've managed to moved to the default auto_reverse flag and rid of this field, functional test seems ok, I am adding ore to test entries